CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+### Added
+- added verification of the checksums for downloaded terraform files.
```

Comment:
Do we have any automation around this? 🤔

Reply:
I don't think so - probably not something for this PR, right?

Reply:
Looks like yes. Let's remove this bit for now.
Ww'll need to investigate options on changelog automation later.

Reply:
Removed this entry